setup.py

```diff
@@ -3,9 +3,13 @@
 with open("README.md", "r") as fh:
     long_description = fh.read()
 
+version = {}
+with open("./vortexasdk/version.py") as fp:
+    exec(fp.read(), version)
```

Comment:
could you replace this with `from vortexasdk.version import __version__`?

Reply:
Not 100% sure whether this works in a setup.py script

Reply:
So interestingly we can't do this (I tried). 

The reason we can't is because when executes `setup.py` in order to create the `vortexasdk` module. pip hasn't gathered or installed the requirements yet, so attempting to `import vortexasdk.version` crashes.

Reply:
Nice, I figured you might have tried already.  Good to know.